include/ocpp/common/message_queue.hpp

```diff
@@ -242,7 +248,13 @@ template <typename M> class MessageQueue {
             if (remove_next_update_message && element->isTransactionUpdateMessage() &&
                 transaction_message_queue.size() > 1) {
                 EVLOG_debug << "Drop transactional message " << element->initial_unique_id;
-                database_handler->remove_transaction_message(element->initial_unique_id);
+                try {
+                    database_handler->remove_transaction_message(element->initial_unique_id);
+                } catch (const QueryExecutionException& e) {
+                    EVLOG_warning << "Could not delete message from transaction queue: " << e.what();
+                } catch (const std::exception& e) {
```

Comment:
Is there a reason we still have these ones? Just as a "catch all" thing to make sure we don't crash I assume?

Reply:
Yes I tried to keep the `std::exception` where it was already present